modules/reference/pages/feature/jwtSso/examples.adoc

```diff
@@ -31,8 +31,9 @@ You can use a mechanism other than JWT cookies for authentication, as the JWT co
 
 ===  Configure an authentication filter
 
-You can configure an authentication filter to secure requests for accessing protected resources that are used with JWT SSO for authentication.
-The `jwtSso` feature supports authentication filters.
+You can configure an authentication filter to secure requests for accessing protected resources that are configured with JWT SSO for authentication.
```

Comment:
suggested edit

You can configure an authentication filter to specify whether certain requests for protected resources are authenticated with JWT SSO.